src/_redux/main/fileTree/event/slice.ts

```diff
@@ -31,4 +31,12 @@ export const FileEventReducer = undoable(fileEventSlice.reducer, {
   undoType: FileTree_Event_UndoActionType,
   redoType: FileTree_Event_RedoActionType,
   clearHistoryType: FileTree_Event_ClearActionType,
+
+  groupBy: (action, currentState, previousHistory) => {
```

Comment:
What are we grouping here and why?

Reply:
for example, when we create a file and undo
it will delete that created file.
and we will need to remove that events in the history line.
this is for that case.